public/index.html

```diff
@@ -10,9 +10,16 @@
       user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
     -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+<<<<<<< HEAD:visual-music-p5-react-wrapper/public/index.html
```

Comment:
It looks like you had to fix conflicting files. When fixing it you need to remove these lines of code 
`<<<<<<< HEAD:visual-music-p5-react-wrapper/public/index.html` 
`=======`
`>>>>>>> upstream/development:public/index.html`

Reply:
HI Arnas, I checked the file changes, it turns out I did not carefully check. I'll fix it now and delete entire visual-music-p5-react-wrap folder.
upload-song PR took about 10sec and I did pay attention on it but also same time I did PR after updating my entire file structure, so assumed it maybe it's normal. Now I know, thank you.